### ISSUE TITLE
0.9.4: java.io.FileNotFoundException: src/main/resources/js/clientsidescripts.js

### ISSUE THREAD

**Issue by hoxu** [paul-hammant/ngWebDriver#16]
I get the following with 0.9.4:

```
Caused by: java.lang.RuntimeException: java.io.FileNotFoundException: src/main/resources/js/clientsidescripts.js (No such file or directory)
        at com.paulhammant.ngwebdriver.AngularJavaScriptFunctions.<clinit>(AngularJavaScriptFunctions.java:26)
        ... 106 more
Caused by: java.io.FileNotFoundException: src/main/resources/js/clientsidescripts.js (No such file or directory)
        at java.io.FileInputStream.open0(Native Method)
        at java.io.FileInputStream.open(FileInputStream.java:195)
        at java.io.FileInputStream.<init>(FileInputStream.java:138)
        at com.paulhammant.ngwebdriver.AngularJavaScriptFunctions.<clinit>(AngularJavaScriptFunctions.java:24)
        ... 106 more
Tests failed.
```

```
$ atool -l ngwebdriver-0.9.4.jar |grep -i clientsidescripts.js
    23855  11-29-15 13:47   js/clientsidescripts.js
```

https://github.com/paul-hammant/ngWebDriver/blob/efc84396c975cbd961e1f8d9829bb913aaa76227/src/main/java/com/paulhammant/ngwebdriver/ByAngular.java#L21-L25

**Comment by paul-hammant** [comment]
I can see the file within the jar:

    wget http://central.maven.org/maven2/com/paulhammant/ngwebdriver/0.9.4/ngwebdriver-0.9.4.jar
    jar tvf ngwebdriver-0.9.4.jar | grep clientsidescripts
       23855 Sun Nov 29 13:47:22 EST 2015 js/clientsidescripts.js

Which JDK, which OS? Are you running ngWebDriver in a classloader?

In fact, can you make a tiny github project where this is reproduced?

**Comment by hoxu** [comment]
Yes, it's in the jar (as can be seen in my atool paste too). Oracle java 1.8.0_66 on OS X El Capitan. I'm using the jar from leiningen.

I'm afraid I don't have time to narrow this down, but hopefully these details are of some use, especially if other people run into this issue as well.

**Comment by paul-hammant** [comment]
Can you do a checkout (I've just made a change), then:

    mvn clean install -DskipTests

Then try ngwebdriver-0.9.5-SNAPSHOT.jar in your Clojure project and report back ?

**Comment by hoxu** [comment]
The FileNotFoundException seems to be fixed now, I got this instead:

     Exception in thread "main" java.lang.ClassNotFoundException: com.paulhammant.ngwebdriver.WaitForAngularRequestsToFinish

Which seems to be from API changes:

```
-WaitForAngularRequestsToFinish.waitForAngularRequestsToFinish(driver);
+new NgWebDriver(driver).waitForAngularRequestsToFinish();
```

I don't have time to upgrade our project properly at this point, but the original issue seems to be fixed with df81320e0e47805f54dd483f5b93714fa925d339, so I think this issue can be closed.

Thanks!
